Library/test-helpers/lib.sh

```diff
@@ -2006,12 +2005,7 @@ limeconPrepareImage() {
         ARGS="--volume /var/tmp/keylime_sources:/mnt/keylime_sources:z"
     fi
 
-    #set up for ssh access
```

Comment:
Probably that should be keep it there regarding to containers for keylime ansible roles testing or rework it.